integration-tests/tests/network/runner.rs

```diff
@@ -911,6 +927,7 @@ where
               flag: Arc<AtomicBool>,
               _ctx: &mut Context<WaitOrTimeoutActor>,
               _runner: Addr<Runner>| {
+            debug!(target: "network", "runner.rs wait_for predicate");
```

Comment:
(general comment): let's make sure that these debug logs are not too spammy - otherwise we risk that one log can completely overwhelm the whole log-file